pkg/cmd/operator.go

```diff
@@ -40,16 +41,18 @@ func newCmdOperator() (*cobra.Command, *operatorCmdOptions) {
 	cmd.Flags().Int32("health-port", 8081, "The port of the health endpoint")
 	cmd.Flags().Int32("monitoring-port", 8080, "The port of the metrics endpoint")
 	cmd.Flags().Bool("leader-election", true, "Use leader election")
+	cmd.Flags().String("leader-election-id", platform.OperatorLockName, "Use the given id as leader election discriminator")
 
 	return &cmd, &options
 }
 
 type operatorCmdOptions struct {
-	HealthPort     int32 `mapstructure:"health-port"`
-	MonitoringPort int32 `mapstructure:"monitoring-port"`
-	LeaderElection bool  `mapstructure:"leader-election"`
+	HealthPort       int32  `mapstructure:"health-port"`
+	MonitoringPort   int32  `mapstructure:"monitoring-port"`
+	LeaderElection   bool   `mapstructure:"leader-election"`
+	LeaderElectionID string `mapstructure:"leader-election-id"`
 }
 
 func (o *operatorCmdOptions) run(_ *cobra.Command, _ []string) {
-	operator.Run(o.HealthPort, o.MonitoringPort, o.LeaderElection)
+	operator.Run(o.HealthPort, o.MonitoringPort, o.LeaderElection, o.LeaderElectionID)
```

Comment:
We need to check if that parameter exists, and, if not, fallback to `platform.OperatorLockName` (which I'd call at this stage as `platform.OperatorDefaultLockName`)